k8s-deployment/encoding-service/values.yaml

```diff
@@ -52,18 +60,22 @@ jobs:
       repository: "data61/anonlink-encoding-service"
       tag: "latest"
 
+global:
+  postgresql:
+    postgresqlDatabase: "postgres"
+    postgresqlUsername: "postgres"
```

Comment:
someone really loves writing 'postgresql'. 
Wouldn't it be more concise like that:
`.Values.global.postgresql.password` etc.

Reply:
Yeah it would but the keys come from the [dependent chart's parameters](https://github.com/bitnami/charts/tree/master/bitnami/postgresql#parameters) not from me.